### ISSUE TITLE
Can Google stop updating the Google Play Billing library ?

### ISSUE THREAD

**Issue by agGitHub** [android/play-billing-samples#518]
We are at the v4 of the lib ; the v1 was itself divided in several 1.x versions, each time with big changes. At each version or subversion, major changes. Both the logic and method names are changed.

The code for the billing lib is very stable on iOS. Why does Google do so many changes ? 

Please make this lib stable. We have no time to spend on this topic, especially when the features we use remain unchanged for the user (one time payment, subscription). No added value to these updates.

Thanks.

**Comment by henrik-lindqvist** [comment]
The billing library has severe threading and security bugs, yet is hardly every updated, maybe once a year. They should update it  more often, fixing the bugs.

**Comment by cartland** [comment]
@henrik-lindqvist to submit bugs or feature requests for the billing library, please share information in the [Google Issue Tracker](https://issuetracker.google.com/issues/new?component=311487&template=1014446).

@agGitHub thanks for the feedback! We're trying to balance improvements to the API with the ease of upgrading. We hear you very clearly that you don't want to make any code changes unless you specifically need new features. We'll pass along this feedback to the right people! Also, if you have any additional feedback about the library, you can file it directly in the Google Issue Tracker.

https://developer.android.com/google/play/billing/contact

**Comment by henrik-lindqvist** [comment]
@cartland I did, about a year ago, and those security issues has still not been fixed, i.e. in 4.1.0 the only update since.

**Comment by cartland** [comment]
@henrik-lindqvist Thanks, can you please share the issue tracker number for the issue you filed?

**Comment by henrik-lindqvist** [comment]
@cartland The most security issues that i've reported are:
https://issuetracker.google.com/issues/190789515
https://issuetracker.google.com/issues/194002577

**Comment by pyman2** [comment]
On the subject of whether it is worth the effort to report bugs to the issue tracker, I too reported an issue in Feb 2021, which has not been fixed.  It was filed as an internal bug and I have heard nothing since.  It sends incorrect subscription information to the app and may (that is, it will if real subscriptions work the same as test subscriptions) allow users to extend their subscriptions for free:  https://issuetracker.google.com/issues/179463140

**Comment by ashughes** [comment]
@cartland I noticed in March that all Google Play Billing issues have been marked as private, so I filed this issue: https://issuetracker.google.com/issues/222726796. I'm hoping this was a mistake, but it's been a few months now and this hasn't even been acknowledged.

As a result, I cannot view the issues that have been posted above or see if an issue that I'm running into has already been filed (along with the status or any workarounds).
